qa/os/src/test/java/org/elasticsearch/packaging/test/DockerTests.java

```diff
@@ -476,16 +480,23 @@ public void test084SymlinkToFileWithInvalidPermissionsIsRejected() throws Except
      * Check that environment variables are translated to -E options even for commands invoked under
      * `docker exec`, where the Docker image's entrypoint is not executed.
      */
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/67097")
-    public void test085EnvironmentVariablesAreRespectedUnderDockerExec() {
+    public void test085EnvironmentVariablesAreRespectedUnderDockerExec() throws Exception {
```

Comment:
So does this address #67097 as well?

Reply:
Yes, there's 2 issues open against this test and I'm hoping this change resolves both.

Reply:
Should we add them to the "closes" description of this issue?

Reply:
Gah, I knew, I _knew_, I'd forget to add those 🤦